test/test_jit.py

```diff
@@ -2613,6 +2613,65 @@ def forward(self, x):
         f = io.BytesIO()
         torch.onnx._export(mte, (torch.zeros(1, 2, 3),), f, verbose=False)
 
+    def test_onnx_export_script_module_loop(self):
+        class ModuleToExport(torch.jit.ScriptModule):
+            def __init__(self):
+                super(ModuleToExport, self).__init__()
+
+            @torch.jit.script_method
+            def forward(self, x):
+                for _ in range(100):
+                    x = x + x
+                return x
+
+        mte = ModuleToExport()
+        f = io.BytesIO()
+        torch.onnx._export(mte, (torch.zeros(1, 2, 3),), f, verbose=False)
+
+    def test_onnx_export_script_module_if(self):
+        class ModuleToExport(torch.jit.ScriptModule):
+            def __init__(self):
+                super(ModuleToExport, self).__init__()
+
+            @torch.jit.script_method
+            def forward(self, x):
+                if torch.sum(x) > 0:
+                    x = torch.neg(x)
+                return x
+
+        mte = ModuleToExport()
+        f = io.BytesIO()
+        torch.onnx._export(mte, (torch.zeros(1, 2, 3),), f, verbose=False)
```

Comment:
These tests don't seem to check the output is sane. Is it possible to do better?

Reply:
I think there's two options here:

1) Expect files
2) Backend checks in `onnx-fb-universe`

Should I implement option 1? Option 2 will have to come in a separate PR